docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -53,50 +64,53 @@ After the shebang and UTF-8 coding, add a `copyright line <https://www.linuxfoun
-.. _ansible_metadata_block:
-
-ANSIBLE_METADATA block
-======================
-
-Since we moved to collections we have deprecated the METADATA functionality, it is no longer required for modules, but it will not break anything if present.
+  * If the existing copyright statement includes a year, do NOT edit the existing copyright year.
 
+* Do NOT modify the existing copyright header without permission from the copyright author.
 
 .. _documentation_block:
 
 DOCUMENTATION block
 ===================
 
-After the shebang, the UTF-8 coding, the copyright line, and the license section comes the ``DOCUMENTATION`` block. Ansible's online module documentation is generated from the ``DOCUMENTATION`` blocks in each module's source code. The ``DOCUMENTATION`` block must be valid YAML. You may find it easier to start writing your ``DOCUMENTATION`` string in an :ref:`editor with YAML syntax highlighting <other_tools_and_programs>` before you include it in your Python file. You can start by copying our `example documentation string <https://github.com/ansible/ansible-documentation/blob/devel/examples/DOCUMENTATION.yml>`_ into your module file and modifying it. If you run into syntax issues in your YAML, you can validate it on the `YAML Lint <http://www.yamllint.com/>`_ website.
+After the shebang, the UTF-8 coding, the copyright line, and the license section comes the ``DOCUMENTATION`` block. Ansible's online module documentation is generated from the ``DOCUMENTATION`` blocks in each module's source code.
 
-Module documentation should briefly and accurately define what each module and option does, and how it works with others in the underlying system. Documentation should be written for broad audience--readable both by experts and non-experts.
-    * Descriptions should always start with a capital letter and end with a full stop. Consistency always helps.
-    * Verify that arguments in doc and module spec dict are identical.
-    * For password / secret arguments ``no_log=True`` should be set.
-    * For arguments that seem to contain sensitive information but **do not** contain secrets, such as "password_length", set ``no_log=False`` to disable the warning message.
-    * If an option is only sometimes required, describe the conditions. For example, "Required when I(state=present)."
-    * If your module allows ``check_mode``, reflect this fact in the documentation.
+The ``DOCUMENTATION`` block must be valid YAML. To make it easier:
 
-To create clear, concise, consistent, and useful documentation, follow the :ref:`style guide <style_guide>`.
+* Start by copying our `example documentation string <https://github.com/ansible/ansible-documentation/blob/devel/examples/DOCUMENTATION.yml>`_.
+* Write the block in an :ref:`editor with YAML syntax highlighting <other_tools_and_programs>` before you include it in your Python file.
+* If you run into syntax issues you do not know how to solve, use the `YAML Lint <http://www.yamllint.com/>`_ website to validate it.
```

Comment:
```suggestion
* If you run into syntax issues that are difficult to resolve, use the `YAML Lint <http://www.yamllint.com/>`_ website to help validate the YAML.
```

Reply:
I would soften the language here to avoid any inference that authors are unable to resolve syntax issues on their own. "you do not know how to" might come across as talking down to the reader a little.